configs/metals.json

```diff
@@ -1,8 +1,7 @@
 {
   "index_name": "metals",
   "start_urls": [
-    "http://scalameta.org/metals/docs/",
-    "http://scalameta.org/metals/docs/installation-contributors.html"
+    "http://scalameta.org/metals/"
   ],
   "stop_urls": [],
   "selectors": {
```

Comment:
The HTML structure of the page may also have changed due to upgrading https://docusaurus.io/, the site generator we use. It might be good to re-generate these selectors.